corehq/ex-submodules/soil/exceptions.py

```diff
@@ -1,4 +1,6 @@
 
 
 class TaskFailedError(Exception):
-    pass
+    def __init__(self, errors, *args, **kwargs):
```

Comment:
will making `errors` required not cause [empty args creation](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/soil/util.py#L40) to fail?